Remotion/Web/Core/ExecutionEngine/WxePageInfo.cs

```diff
@@ -288,7 +290,11 @@ public void OnPreRenderComplete ()
 
       _page.ClientScript.RegisterHiddenField (_page, WxeHandler.Parameters.WxeFunctionToken, wxeContext.FunctionToken);
       _page.ClientScript.RegisterHiddenField (_page, WxePageInfo.ReturningTokenID, null);
-      _page.ClientScript.RegisterHiddenField (_page, WxePageInfo.PostBackSequenceNumberID, wxeContext.PostBackID.ToString ());
+      int nextPostBackID = wxeContext.PostBackID + 1;
+      _page.ClientScript.RegisterHiddenField (_page, WxePageInfo.PostBackSequenceNumberID, nextPostBackID.ToString ());
+
+      if (SafeServiceLocator.Current.GetInstance<IRenderingFeatures>().EnableDiagnosticMetadata)
+        _page.ClientScript.RegisterHiddenField (_page, WxePageInfo.DmaPostBackSequenceNumberID, wxeContext.PostBackID.ToString());

```

Comment:
make a second variable currentPostBackID remove code duplication and make it clear that these two are intentionally different usages of the postback ID. Otherwise, it would look like it's an oversight.